src/lib/theme.js

```diff
@@ -0,0 +1,28 @@
+const BASE_MEASURE = 8
+
+const spacing = (n) => n * BASE_MEASURE
+
+const shadow = () => ({
```

Comment:
I guess you're planning to parameterize this at some point, I assume you wouldn't implement this as a function otherwise. Am I right?

Reply:
Yes, it will have levels of shadow something like elevation level of the surface.

Reply:
ok that makes sense